src/main/java/application/MenuController.java

```diff
@@ -147,7 +144,7 @@ public EntityDetailListResponse getDetails(@RequestBody SessionNavigationBean se
             if (detail == null) {
                 throw new RuntimeException("Tried to get details while not on a case list.");
             }
-            return new EntityDetailListResponse(detail);
+            return LocationRelevantResponseBean.setLocationNeeds(new EntityDetailListResponse(detail), menuSession);
```

Comment:
This block is only for persistent case tiles, which I don't think would contain `here()`? Though I could be wrong

Reply:
I mean it's just up to us whether we want to support it in that place. My perspective is that it would be confusing to support `here()` in some details but not others, so I'd rather be consistent.